src/cli.js

```diff
@@ -27,11 +27,15 @@ function secondsToMilliseconds(seconds) {
   return seconds * 1000;
 }
 
-function getFlatCache() {
+function getFlatCache(ttl) {
+  let cache;
   if (process.env.V8R_CACHE_NAME) {
-    return flatCache.load(process.env.V8R_CACHE_NAME);
+    cache = new FlatCache({ cacheId: process.env.V8R_CACHE_NAME, ttl: ttl });
+  } else {
+    cache = new FlatCache({ cacheId: "v8rv2", cacheDir: CACHE_DIR, ttl: ttl });
```

Comment:
I'm changing the name of the cache here because:

- The format of the cache file changes between flat-cache 5 and flat-cache 6
- The object I'm storing in each cache key has also slightly changed

I think the simplest way to force an invalidation is to change the name of the cache file in this version.